src/value.rs

```diff
@@ -638,6 +638,12 @@ impl<'a> From<&'a str> for Value {
     }
 }
 
+impl From<Vec<u8>> for Value {
```

Comment:
Can you use `From<Data>` here? `From<Vec<u8>>` could make sense for both array or data and I'd rather avoid the ambiguity.